source/playbooks/work-with-tasks.rst

```diff
@@ -8,13 +8,13 @@ In some workflows, there are time constraints on tasks and others may have more
 
 To assign a due date to a task, select the **Toggle Run Details** icon to open the **Run Details** screen. Hover over the task you’d like to edit and select the calendar icon to assign a due date. Due dates can be used to sort tasks in the run overview.
 
-When a due date is assigned to a task, and the task is overdue or due today, a reminder is added to the Playbooks daily digest along with tasks that don't have an assigned due date. As tasks are completed, they're removed from the daily digest reminders. You can refresh the list of assigned tasks at any time using the ``/playbooks to-do`` slash command.
+When a due date is assigned to a task, and the task is overdue or due today, a reminder is added to the Playbooks daily digest along with tasks that don't have an assigned due date. As tasks are completed, they're removed from the daily digest reminders. You can refresh the list of assigned tasks at any time using the ``/playbooks todo`` slash command.
 
 Due dates can be entered in text (e.g., "two minutes ago") or numerically (e.g., "15 March 2023").
 
 Task inbox
 ----------
 
-In addition to the daily digest, you also have access to a task inbox. To view it, open the playbooks tab. 
+In addition to the daily digest, you also have access to a task inbox. The task inbox provides you with a cross-run overview of the tasks you're accountable for, sorted by due date. You can access and complete each task directly, without having to visit the individual runs.
 
-In the header, next to your profile photo, select the message icon. A list of every task assigned to you from every run that's in progress is displayed. You can check off, change due dates, and reassign tasks in this view.
+To view it, open the playbooks tab. In the header, next to your profile photo, select the message icon. A list of every task assigned to you from every run that's in progress is displayed. You can check off, change due dates, and reassign tasks in this view.
```

Comment:
nit: it's not a message icon but a tasks list icon. TBF it's super tiny and easy to misunderstand